qa/tasks/deepsea.py

```diff
@@ -1465,7 +1467,28 @@ def begin(self):
                 "config dictionary may contain only one key. "
                 "You provided ->{}<- keys ({})".format(len(config_keys), config_keys)
                 )
-        role_spec, repositories = self.config.items()[0]
+        role_spec, paramdict = self.config.items()[0]
+        if not isinstance(paramdict, dict):
+            paramdict = {'tries': reboot_tries}
```

Comment:
Is it a common practice to override parameters if they have different type? what if it is number value and user will not be aware that he entered something wrong.

Reply:
`reboot_tries` is already defined on top of the file